MISC/ClearWeatherTypeNowPersistNetwork.md

```diff
@@ -1,11 +1,12 @@
 ---
 ns: MISC
+aliases: ["_CLEAR_WEATHER_TYPE_OVERTIME_PERSIST"]
```

Comment:
Hey there, alias is supposed to be `0x0CF97F497FE7D048` i.e.

```
aliases: ["0x0CF97F497FE7D048"]
```

Not sure why the alias is named that 🤔

Reply:
Should be good